projects/plugins/jetpack/extensions/blocks/anchor-fm/templates/index.js

```diff
@@ -76,27 +76,31 @@ function podcastSection( { episodeTrack, feedUrl, coverImage } ) {
 }
 
 function podcastSummarySection( { episodeTrack } ) {
-	return [
-		'core/group',
-		{},
+	const sectionBlocks = [
 		[
-			[
-				'core/heading',
-				{
-					level: 3,
-					content: 'Summary',
-					placeholder: __( 'Podcast episode title', 'jetpack' ),
-				},
-			],
-			[
-				'core/paragraph',
-				{
-					placeholder: __( 'Podcast episode summary', 'jetpack' ),
-					content: episodeTrack.description_html,
-				},
-			],
+			'core/heading',
+			{
+				level: 3,
+				content: 'Summary',
```

Comment:
Can we translate it here?

```suggestion
				content: __( 'Summary', 'jetpack' ),
```

Reply:
Nice catch!